languages/go/INSTRUCTIONS.md

```diff
@@ -0,0 +1,150 @@
+# Instructions
+
+This guide is for developers who want to use the Bitwarden Go SDK module in their own Go projects. Please see the main [README](./README.md) and [example.go](./example/example.go) file for examples.
+
+## Supported Targets
+
+The Bitwarden Go SDK module utilizes FFI calls to the Bitwarden Rust SDK via [cgo](https://pkg.go.dev/cmd/cgo). The module supports the following statically linked targets:
+
+- Linux `x86-64` & `arm64`
+- macOS `x86-64` & `arm64`
+- Windows `x86-64`
+
+## Linux
+
+### Prerequisites
+
+- [Go](https://go.dev/dl)
+- A C toolchain
+
+We recommend the [MUSL toolchain](https://musl.libc.org). You can install this on most debian based systems with:
+
+```shell
+sudo apt install musl-tools
+```
+
+### Set Go Environment Info
+
+#### Enable cgo
+
+```shell
+go env -w CGO_ENABLED=1
+```
+
+#### Set the C compiler
+
+```shell
+go env -w CC=musl-gcc
+```
+
+#### Verify
+```shell
+go env
+```
+
+### Install the Bitwarden Go SDK
+
+#### Adding the Module
+
+```shell
+go get github.com/bitwarden/sdk-go
+```
+
+#### Build
+
+```shell
+go build -ldflags '-linkmode external -extldflags "-static -Wl,-unresolved-symbols=ignore-all"'
+```
+
+## Mac
```

Comment:
```suggestion
## macOS
```

Reply:
Fixed in https://github.com/bitwarden/sdk/pull/861/commits/92b64f40c48540efda0a46405368cd5da5866c59